arelle/plugin/validate/NL/DisclosureSystems.py

```diff
@@ -0,0 +1,6 @@
+DISCLOSURE_SYSTEM_NT16 = 'NT16'
+DISCLOSURE_SYSTEM_NT17 = 'NT17'
+DISCLOSURE_SYSTEM_NT18 = 'NT18'
+DISCLOSURE_SYSTEM_NT16_PREVIEW = 'NT16-preview'
+DISCLOSURE_SYSTEM_NT17_PREVIEW = 'NT17-preview'
+DISCLOSURE_SYSTEM_NT18_PREVIEW = 'NT18-preview'
```

Comment:
I think we should drop the non preview disclosure systems for now. I don't want any users to think that this is ready to depend on.

I also think we should name the variables `DISCLOSURE_SYSTEM_NT16`, `DISCLOSURE_SYSTEM_NT17`, and `DISCLOSURE_SYSTEM_NT18`, but have the text content `NT16-preview`, `NT17-preview`, and `NT18-preview`. That way when we decide to move this out of preview status we can just update the text values without touching any of the rules.